autoreduce_qp/queue_processor/reduction/runner.py

```diff
@@ -123,7 +139,11 @@ def main():
     Additionally, the resulting Message is written to a temporary file which the
     parent process reads back to mark the result of the reduction run in the DB.
     """
-    data, temp_output_file, run_name = sys.argv[1], sys.argv[2], sys.argv[3]
+    data, run_name = sys.argv[1], sys.argv[2]
+    if len(sys.argv) > 3:
+        temp_output_file = sys.argv[3]
+    else:
+        temp_output_file = None
```

Comment:
Same for this as comment above - since no `temp_output_file` is expected anymore it will always end up at `/output/output.txt`